aplus/settings.py

```diff
@@ -29,6 +29,7 @@
 SITEWIDE_ALERT_TEXT = None
 SITEWIDE_ADVERT = None
 BRAND_NAME = 'A+'
+BRAND_DESCRIPTION = 'Virtual Learning Environment at Aalto University Department of Computer Science'
```

Comment:
I think the default settings should avoid very Aalto-specific contents like this description that mentions "Aalto University". Should the default be just "Virtual learning environment"? We should also document the settings that should be overridden in the `local_settings.py` of a real production installation. Is it enough to add them to the `local_settings.example.py` file?

Reply:
>  Is it enough to add them to the local_settings.example.py file?

I think it is, but I'm not sure. I personally read that file first, so if it's documented somewhere else, then we need to duplicate the information. Most important values are mentioned in the deployment document.